packages/@aws-cdk/aws-pipes-alpha/test/pipe.test.ts

```diff
@@ -449,28 +436,314 @@ describe('Pipe', () => {
+          },
+        },
+      });
+    });
+
+    it('should pass along s3 log configuration for cross-account bucket', () => {
+      // WHEN
+      const bucket = new Bucket(stack, 'LogBucket');
+      const s3LogDestination = new S3LogDestination({
+        bucket,
+        bucketOwner: '111111111111',
+        prefix: 'mike',
+        outputFormat: S3OutputFormat.JSON,
+      });
+
+      new Pipe(stack, 'TestPipe', {
+        pipeName: 'TestPipe',
+        source,
+        target,
+        logLevel: LogLevel.ERROR,
+        logIncludeExecutionData: [IncludeExecutionData.ALL],
+        logDestinations: [s3LogDestination],
+      });
+
+      const template = Template.fromStack(stack);
+
+      // THEN
+      template.hasResource('AWS::Pipes::Pipe', {
+        Properties: {
+          LogConfiguration: {
+            S3LogDestination: {
+              BucketName: {
+                Ref: 'LogBucketCC3B17E8',
+              },
+              BucketOwner: '111111111111',
+              Prefix: 'mike',
+              OutputFormat: 'json',
+            },
+            Level: 'ERROR',
+            IncludeExecutionData: ['ALL'],
+          },
+        },
+      });
+    });
+
+    it('should pass along bucketOwner for imported bucket', () => {
+      // WHEN
+      const bucket = Bucket.fromBucketName(stack, 'ImportedBucket', 'MyTestBucket');
+      const s3LogDestination = new S3LogDestination({
+        bucket,
+        prefix: 'mike',
+        outputFormat: S3OutputFormat.JSON,
+      });
+
+      new Pipe(stack, 'TestPipe', {
+        pipeName: 'TestPipe',
+        source,
+        target,
+        logLevel: LogLevel.ERROR,
+        logIncludeExecutionData: [IncludeExecutionData.ALL],
+        logDestinations: [s3LogDestination],
+      });
+
+      const template = Template.fromStack(stack);
+
+      // THEN
+      template.hasResource('AWS::Pipes::Pipe', {
+        Properties: {
+          LogConfiguration: {
+            S3LogDestination: {
+              BucketName: 'MyTestBucket',
+              BucketOwner: '123456789012',
+              Prefix: 'mike',
+              OutputFormat: 'json',
+            },
+            Level: 'ERROR',
+            IncludeExecutionData: ['ALL'],
+          },
+        },
+      });
+    });
+
+    it('should allow write to S3 bucket with pipe role', () => {
+      // WHEN
+      const bucket = new Bucket(stack, 'LogBucket');
+      const s3LogDestination = new S3LogDestination({ bucket });
+
+      new Pipe(stack, 'TestPipe', {
+        pipeName: 'TestPipe',
+        source,
+        target,
+        logDestinations: [s3LogDestination],
+      });
+
+      const template = Template.fromStack(stack);
+
+      // ASSERT
+      template.hasResource('AWS::IAM::Policy', {
+        Properties: {
+          Roles: [{
+            Ref: 'TestPipeRole0FD00B2B',
+          }],
+          PolicyDocument: {
+            Statement: [{
+              Action: [
+                's3:DeleteObject*',
```

Comment:
Similar callout to the S3 `grantWrite()`, is `s3:DeleteObject*` a necessary permission for storing the logs?